WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackConnectionWebViewModel.swift

```diff
@@ -57,6 +61,18 @@ final class JetpackConnectionWebViewModel: AuthenticatedWebViewModel {
         return .allow
     }
 
+    func decidePolicy(for response: URLResponse) async -> WKNavigationResponsePolicy {
+        guard let urlResponse = response as? HTTPURLResponse, urlResponse.statusCode != 404 else {
+            return .allow
+        }
+        await MainActor.run { [weak self] in
+            guard let self else { return }
+            self.isFailure = true
```

Comment:
Super nit: 

My understanding is we introduced `isFailure` to allow dismissal of the view. 

Could we instead keep using `isCompleted` and set it as `true`? Maybe we can rename it into something like `isProcessing`. 

Please feel free to keep the new `isFailure` variable if you think it adds to readability.

Reply:
That makes sense, I unified `isCompleted` and `isFailure` into `shouldIgnoreDismissalHandling` for clarity in b12441481235efea7b46894c4f3b8368454ecef0.